packages/components/src/components/multi-select/multi-select/multi-select-styles.ts

```diff
@@ -0,0 +1,194 @@
+import type { BreakpointCustomizable, Theme } from '../../../types';
+import { buildResponsiveStyles, getCss } from '../../../utils';
+import {
+  addImportantToEachRule,
+  addImportantToRule,
+  getHiddenTextJssStyle,
+  getThemedColors,
+  getTransition,
+  hostHiddenStyles,
+  hoverMediaQuery,
+} from '../../../styles';
+import {
+  borderRadiusSmall,
+  borderWidthBase,
+  fontLineHeight,
+  spacingStaticMedium,
+  spacingStaticSmall,
+  spacingStaticXSmall,
+  textSmallStyle,
+} from '../../../../../utilities/projects/utilities';
```

Comment:
Should be imported from `@porsche-design-system/utilities-v2`. Might be good to prevent this by extending `.eslintrc.js`.